omod-2.0/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/converter/openmrs2_0/VisitConfigurationConverter2_0.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.converter.openmrs2_0;
+
+import org.openmrs.VisitType;
+import org.openmrs.annotation.Handler;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.api.RestService;
+import org.openmrs.module.webservices.rest.web.representation.FullRepresentation;
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingConverter;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.response.ConversionException;
+import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_9.VisitTypeResource1_9;
+import org.openmrs.module.webservices.rest.web.v1_0.wrapper.VisitConfiguration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Handler(supports = VisitConfiguration.class, order = 0)
+public class VisitConfigurationConverter2_0 extends BaseDelegatingConverter<VisitConfiguration> {
+
+	@Override
+	public DelegatingResourceDescription getRepresentationDescription(Representation rep) {
+		DelegatingResourceDescription description = new DelegatingResourceDescription();
+		description.addProperty("encounterVisitsAssignmentHandler");
+		description.addProperty("enableVisits");
+		description.addProperty("startAutoCloseVisitsTask");
+		if (rep instanceof FullRepresentation) {
+			description.addProperty("visitTypesToAutoClose");
+			return description;
+		}
+		return null;
```

Comment:
So you are returning null for everything expect full representation?

Reply:
My fault :) https://github.com/openmrs/openmrs-module-webservices.rest/pull/489/commits/2f898eb0ac4b53d31fe2215530ceeaf69b9a5ef3